adaptive/runner.py

```diff
@@ -269,7 +276,8 @@ class BlockingRunner(BaseRunner):
         the learner as its sole argument, and return True when we should
         stop requesting more points.
     executor : `concurrent.futures.Executor`, `distributed.Client`,\
-               `mpi4py.futures.MPIPoolExecutor`, or `ipyparallel.Client`, optional
+               `mpi4py.futures.MPIPoolExecutor`, `ipyparallel.Client` or\
+               `loky.get_reusable_executor`, optional
```

Comment:
nitpick: `loky.get_reusable_executor` does not name a type, but maybe that's ok

Reply:
Right, this was a conscious choice, the type is `loky.reusable_executor._ReusablePoolExecutor` but IIUC one can't instantiate it directly. Therefore, I think it's more useful to write `loky.get_reusable_executor`.